### PR TITLE
community: updated openai prices in mapping

### PR DESCRIPTION
**Pull request by Sssanek**
- **Description:** there are january prices update for chatgpt [blog](https://openai.com/blog/new-embedding-models-and-api-updates), also there are updates on their website on page [pricing](https://openai.com/pricing)
- **Issue:** N/A

### CODE REVIEW

**Comment by Sssanek** [comment]
image

**Review by hwchase17 — APPROVED** [review]
thanks

File: libs/community/langchain_community/callbacks/openai_info.py [hunk 1]
```diff
@@ -25,15 +25,17 @@
     "gpt-4-vision-preview-completion": 0.03,
     "gpt-4-1106-preview-completion": 0.03,
     # GPT-3.5 input
-    "gpt-3.5-turbo": 0.0015,
+    "gpt-3.5-turbo": 0.0005,
```
Comment (baskaryan) [inline libs/community/langchain_community/callbacks/openai_info.py:28]: have base gpt-3.5-turbo prices changed? till 2/16 it still points at -0613 not -0125
Screenshot 2024-02-05 at 6 11 14 PM

Comment (Sssanek) [reply libs/community/langchain_community/callbacks/openai_info.py:28]: yep, for 2 days gpt-3.5-turbo will be -0613 with price 0.0015

Comment (Sssanek) [reply libs/community/langchain_community/callbacks/openai_info.py:28]: but then it will be -0125 with new price

Comment (Sssanek) [reply libs/community/langchain_community/callbacks/openai_info.py:28]: ok! now its time!

Comment (baskaryan) [reply libs/community/langchain_community/callbacks/openai_info.py:28]: what makes you say that, isn't feb 16 the flip date?

**Review by baskaryan — APPROVED** [review]

File: libs/community/langchain_community/callbacks/openai_info.py (hunk 1 above)
Comment (Sssanek) [reply libs/community/langchain_community/callbacks/openai_info.py]: there is 
Customers using the pinned gpt-3.5-turbo model alias will be automatically upgraded from gpt-3.5-turbo-0613 to gpt-3.5-turbo-0125 two weeks after this model launches.

**Comment by JabirSS** [comment]
How about updates for the new gpt4 and embeddings pricings? Any plan to fix those?

**Pull request merged**